backends/cdc-postgres/cdc-postgres.go

```diff
@@ -0,0 +1,61 @@
+package cdc_postgres
+
+import (
+	"crypto/tls"
+	"github.com/batchcorp/plumber/cli"
+	"github.com/batchcorp/plumber/printer"
+	"github.com/jackc/pgx"
+	"github.com/sirupsen/logrus"
+)
+
+/*
+Example output
```

Comment:
Do we track the `xid` as well somewhere?

Reply:
Whoops, added